exercises/1-the-manual-menace/README.md

```diff
@@ -212,13 +212,13 @@ echo "zsh" >> ~/.bashrc
 cd enablement-ci-cd
 ```
 
-11.  With the configuration in place, install the OpenShift Applier dependency
+11. With the configuration in place, install the OpenShift Applier dependency
 
 ```bash
 ansible-galaxy install -r requirements.yml --roles-path=roles
 ```
 
-12. Apply the inventory by logging into OpenShift on the terminal and running the playbook as follows (<CLUSTER_URL> should be replaced with the one you've been provided by the instructor). Accept any insecure connection warning(s) 👍:
+12. Apply the inventory by logging into OpenShift on the terminal and running the playbook as follows (`<CLUSTER_URL>` should be replaced with the one you've been provided by the instructor). Accept any insecure connection warning(s) 👍:
```

Comment:
Without the ticks the stuff inside `<>` doesn't render...
<img width="831" alt="Screenshot 2019-09-21 at 13 20 44" src="https://user-images.githubusercontent.com/1270328/65372593-a2f72880-dc72-11e9-89a3-9cdf925f9f3a.png">